bitrise/bitrise.go

```diff
@@ -220,6 +243,118 @@ func (app App) StartBuild(workflow string, buildParams json.RawMessage, buildNum
 	return response, nil
 }
 
+// GetBuildArtifacts ...
+func (app App) GetBuildArtifacts(buildSlug string) (buildArtifactsResponse BuildArtifactsResponse, err error) {
```

Comment:
`buildArtifactsResponse` is not used anywhere

Reply:
It is recommended to avoid named returns. Please remove them.

Reply:
`(app App) GetBuildArtifacts` -> the layers of the system structure is not matching to the actual one. The structure looks somehow like: App -> Build -> Artifact but as you have the method attached to the app this way it looks like: App -> Artifact

When the step runs, and waiting for the result, it is always returning a build(because of it's status we need to request a build info), we could call GetArtifacts on that build model.

Reply:
> `buildArtifactsResponse` is not used anywhere

it is used here but by a different name

https://github.com/bitrise-steplib/bitrise-step-build-router-wait/pull/9/files#diff-7ddfb3e035b42cd70649cc33393fe32cR64